1-js/04-object-basics/04-object-methods/article.md

```diff
@@ -190,23 +190,23 @@ function sayHi() {
 }
 
 *!*
-// use the same function in two objects
+// usa la misma función en dos objetos
 user.f = sayHi;
 admin.f = sayHi;
 */!*
 
-// these calls have different this
-// "this" inside the function is the object "before the dot"
+// estos llamados tienen diferente "this"
+// "this" dentro de la función es el objeto "delante del punto"
```

Comment:
```suggestion
// "this" dentro de la función es el objeto "delante del punto"
```
¿Yo estoy mal? ¿Es mi escasa comprensión del inglés? ¿Por qué "before the dot" es "delante del punto" si el texto en inglés concuerda con el ejemplo en qué `this` es el objeto antes del punto?

 `this == user` y `user` está antes del punto